### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by ben519** [rehype-pretty/rehype-pretty-code#192] [General]
Hey, rehype-pretty-code is an important part of my Markdown-based writing platform, [Scipress.io](https://www.scipress.io/). So, I was hoping I could sponsor this project with a monetary donation in exchange for you placing my logo and a link to my site on the main README. I suspect some of your users would be interested in seeing how I've used rehype-pretty-code in my project.

Let me know if we can work out a deal. Thanks!

**Comment by o-az** [comment]
@ben519 thank you for your offer. I’m currently the sole maintainer and my situation does not permit me to accept any donations or any form of pay outside employment. However we might be able to make this work if you are willing to do a donation to some other open source project that I benefit from. I don’t have any in mind but can come up with a few. 

Thanks!

**Reply by ben519** [reply]
Can you elaborate on "my situation does not permit me to accept any donations or any form of pay outside employment"? What exactly is the issue? I don't want to get you in trouble :)
